src/components/Pokemon.js

```diff
@@ -0,0 +1,46 @@
+import React, {useState, useEffect} from 'react';
+import axios from "axios";
+import './Pokemon.css';
+
+const Pokemon = ({endpoint}) => {
+    const [pokemon, setPokemon] = useState('');
```

Comment:
Ik zou liever als default value `undefined` gebruiken, omdat het type van de waarde nooit een string zal zijn.
```suggestion
    const [pokemon, setPokemon] = useState();
```